Dockerfile

```diff
@@ -1,26 +1,24 @@
-# Use an official Python runtime as a parent image
-FROM python:3.10-slim
-# Set environment variables
-ENV PYTHONDONTWRITEBYTECODE 1
-ENV PYTHONUNBUFFERED 1
+# Use Rust official image as base
+FROM rust:1.82.0-slim-bullseye
 
-# app vars
-ENV STORAGE_DIRECTORY /data
-ENV DB_FILE /data/data.db
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
```

Comment:
Where is the storage directory? is the db file hardcoded we should follow similar style as previous

Reply:
will add the directory 
for the db we can specify it to use the DB_FILE ENV variable to create the sqlite file 
the changes would most likely contain extra code to get the env variable and creation of parent path 
i.e /data